print/Cargo.toml

```diff
@@ -19,3 +19,6 @@ defmt-decoder = { version = "=0.3.9", path = "../decoder", features = [
     "unstable",
 ] }
 log = "0.4"
+notify = "6.1.1"
+tokio = { version = "1", features = ["full"] }
```

Comment:
I share the concern of @jonathanpallant to add tokio as a dependency. Especially with full features. You probably can limit the features.

But otherwise I think a background thread to watch for changes sounds like a more lightweight solution.

Reply:
Fair enough. I think my issue with using a background thread is that in the case of an ELF file change, I'll need to stop the read call on source (`let (n, eof) = source.read(&mut buf)?;`) as soon as possible to be able to reload the ELF file. 
I'm not sure that that is possible with using a timeout